File: program/find-area-of-a-cuboid/FindAreaOfACuboid.js
function cuboidArea(Length,Width,Height){    //Cuboid Area function
    return 2 * (Length * Width + Length * Height + Width * Height);
}

let Length = 2;
let Width = 3;
let Height = 4;
console.log(cuboidArea(Length,Width,Height));   //52

File: program/find-area-of-a-cuboid/findAreaOfACuboid.scala
object HelloWorld {
	def main(args: Array[String]): Unit = {
	  val length = scala.io.StdIn.readDouble();
	  val width = scala.io.StdIn.readDouble();
	  val height = scala.io.StdIn.readDouble();
	  val area = 2 * (length * width + length * height + width * height);
	  println("area of cuboid: " +area);
	}
}
